appveyor.yml

```diff
@@ -0,0 +1,91 @@
+# packcheck-0.4.2
+# You can use any of the options supported by packcheck as environment
+# variables here.  See https://github.com/composewell/packcheck for all
+# options and their explanation.
+branches:
+  only:
+    - master
+
+environment:
+    # ------------------------------------------------------------------------
+    # Global options, you can use these per build as well
+    # ------------------------------------------------------------------------
+  global:
+    # ------------------------------------------------------------------------
+    # Common options
+    # ------------------------------------------------------------------------
+    GHC_OPTIONS: "-Werror"
+    CABAL_REINIT_CONFIG: "y"
+    LC_ALL: "C.UTF-8"
+
+    # ------------------------------------------------------------------------
+    # What to build
+    # ------------------------------------------------------------------------
+    # DISABLE_TEST: "y"
+    # DISABLE_BENCH: "y"
+    # DISABLE_DOCS: "y"
+    DISABLE_SDIST_BUILD: "y"
+    # DISABLE_DIST_CHECKS: "y"
+
+    # ------------------------------------------------------------------------
+    # stack options
+    # ------------------------------------------------------------------------
+    # Note requiring a specific version of stack using STACKVER may fail due to
+    # github API limit while checking and upgrading/downgrading to the specific
+    # version.
+    #STACKVER: "1.6.5"
+    STACK_UPGRADE: "y"
+    RESOLVER: "lts-20.14"
```

Comment:
Should specify a STACK_YAML to use extra deps - see other (streamly) appveyor CIs .